cmd/oapi-codegen/oapi-codegen.go

```diff
@@ -125,3 +134,46 @@ func splitCSVArg(input string) []string {
 	}
 	return args
 }
+
+func loadTemplateOverrides(templatesDir string) (map[string]string, error) {
+	var (
+		knownTemplates = map[string]interface{}{
```

Comment:
I don't like having a hard coded file list - it's another place to forget to update.

We also don't really need to load all templates either, since the Chi and Echo stuff is mutually exclusive, for example.

It's a bummer that that the templates compiler I'm using doesn't export the file map, then this would be much nicer!

Can we simply enumerate the files in the templates directory? - if the filename matches, good, it'll override the original, if it doesn't match, you get no behavior change.

Reply:
I've just pushed a follow-up commit (a9a410b) that implements your suggestion, WDYT?